.danger.php

```diff
@@ -169,190 +110,92 @@ function (Context $context): void {
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                $files = $context->platform->pullRequest->getFiles();
-
-                $bcChange = $files->matches('.bc-exclude.php')->count() > 0;
-
-                if (!$bcChange) {
-                    return;
-                }
-
-                $context->platform->addLabels('bc_exclude_php');
-            },
-        ]
-    ))
-    ->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                if (!preg_match('/(?m)^((WIP:\s)|^(Draft:\s)|^(DRAFT:\s))?(\[[\w.]+]\s)?NEXT-\d*\s-\s\w/', $context->platform->pullRequest->title)) {
-                    $context->failure(sprintf('The title `%s` does not match our requirements. Example: NEXT-00000 - My Title', $context->platform->pullRequest->title));
-                }
-            },
-        ]
-    ))
-    ->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                $labels = $context->platform->pullRequest->labels;
-
-                if (in_array('E2E:skip', $labels, true) || in_array('unit:skip', $labels, true)) {
-                    $context->notice('You skipped some tests. Reviewers be carefully with this');
-                }
-            },
-            function (Context $context): void {
-                $files = $context->platform->pullRequest->getFiles();
-                $hasStoreApiModified = false;
-
-                /** @var File $file */
-                foreach ($files->getElements() as $file) {
-                    if (str_contains($file->name, 'SalesChannel') && str_contains($file->name, 'Route.php') && !str_contains($file->name, '/Test/')) {
-                        $hasStoreApiModified = true;
-                    }
-                }
-
-                if ($hasStoreApiModified) {
-                    $context->warning('Store-API Route has been modified. @Reviewers please review carefully!');
-                    $context->platform->addLabels('Security-Audit Required');
-                }
-            },
-        ]
-    ))
     ->useRule(function (Context $context): void {
         function checkMigrationForBundle(string $bundle, Context $context): void
         {
             $files = $context->platform->pullRequest->getFiles();
 
-            $migrationFiles = $files->filterStatus(File::STATUS_ADDED)->matches('src/Core/Migration/V*/Migration*.php');
-            $migrationTestFiles = $files->filterStatus(File::STATUS_ADDED)->matches('tests/migration/Core/V*/*.php');
+            $migrationFiles = $files->filterStatus(File::STATUS_ADDED)->matches(sprintf('src/%s/Migration/V*/Migration*.php', $bundle));
```

Comment:
if you ever wondered why you get three messages, while adding a new migration :see_no_evil: